tests/attr/test_gradient_shap.py

```diff
@@ -171,9 +174,11 @@ def test_basic_relu_multi_input(self):
 
         ig = IntegratedGradients(model)
         attributions_ig = ig.attribute(inputs, baselines=baselines)
-        self._assert_shap_ig_comparision(attributions, attributions_ig)
+        self._assert_shap_ig_comparision(tuple(attributions), attributions_ig)
```

Comment:
Same here, it would be good to avoid calling tuple here since it's already a tuple.